client/state/reader/streams/selectors/get-reader-stream-next-item.js

```diff
@@ -1,11 +1,9 @@
 /**
- * External dependencies
+ * Internal dependencies
  */
+import getReaderStreamOffsetItem from './get-reader-stream-offset-item';
```

Comment:
Could we use absolute paths when possible? Unless there is a better reason to use the relative ones. 

This also applies for other instances in this PR, so you might want to audit them all.